### ISSUE TITLE
Error Undefined property $category in Moodle 4.0

### ISSUE THREAD

**Issue by danielcifuentesopen** [wiris/moodle-qtype_multianswerwiris#38]
An error is appearing after doing the following steps in Moodle 4.0:

- Create a course
- Go to Course -> Question Bank
- Select "Import" 
- Select Moodle XML format
- Import the following file attached [Test file](https://github.com/wiris/moodle-qtype_multianswerwiris/blob/main/tests/fixtures/testquestion.moodle.xml) This file is located in `multianswerwiris/tests/fixtures/testquestion.moodle.xml` and is used for the Behat tests of the plugin. 
- After this, you should see "Parsing questions from import file.". Click on Continue.
- Now Edit the created question. 
- In the edit view, just click on Save Changes without making any change. 
- The following error appears:

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/114766053/193910647-abfe5bfa-9477-4266-a68a-a05d9489660d.png">

This may be because in multianswerwiris/questiontype.php line 151, `$authorizedquestion->category` is unset. There is a variable that may contain the category which is `$authorizedquestion->questioncategoryid` but this one can also be unset. A possible solution can be something like this:

```
if (isset($authorizedquestion->category)) {
            $question->category = $authorizedquestion->category;
        } elseif (isset($authorizedquestion->questioncategoryid)) {
            $question->category = $authorizedquestion->questioncategoryid;
     }
```

... but I'm not sure in which escenarios this may not work. 

Thanks.

**Comment by ptorrent-at-wiris** [comment]
Hi Daniel, apologies for the long delay. This issue will be solved in our next release.

Best regards

**Comment by ptorrent-at-wiris** [comment]
Hi Daniel, this issue should be solved in our latest release `v4.11.4`. Best,
